apps/aechannel/src/aesc_state_cache.erl

```diff
@@ -194,11 +195,15 @@ move_state_to_persistent(ChId) ->
               ?TAB, [{ #ch{id = key(ChId, '_'), _ = '_'}, [], ['$_'] }]),
     case Found of
         [] -> ok;
-        [#ch{id = {_,PKa}, state = A}, #ch{id = {_,PKb}, state = A}] ->
-            Pch = #pch{id = ChId, pubkeys = [PKa, PKb], state = A},
+        [#ch{id = {_,PKa}, state = A, opts = Oa},
```

Comment:
Can you add some high-level comments to these pattern matches? The detail gets easily lost when scanning the code.